File: ai25btech11021/matgeo/5.12.3/codes/matrix.py
count = 0

# Each entry can be 1, 2, or 3
for a in [1, 2, 3]:
    for b in [1, 2, 3]:
        for c in [1, 2, 3]:
            for d in [1, 2, 3]:
                count += 1
                print(f"Matrix {count}:")
                print(f"{a} {b}")
                print(f"{c} {d}\n")

print("Total matrices =", count)

File: ai25btech11021/matgeo/5.12.3/codes/matrix.c
#include <stdio.h>

int main() {
    int a, b, c, d;
    int count = 0;

    // Each entry can be 1, 2, or 3
    for (a = 1; a <= 3; a++) {
        for (b = 1; b <= 3; b++) {
            for (c = 1; c <= 3; c++) {
                for (d = 1; d <= 3; d++) {
                    count++;
                    printf("Matrix %d:\n", count);
                    printf("%d %d\n", a, b);
                    printf("%d %d\n\n", c, d);
                }
            }
        }
    }

    printf("Total matrices = %d\n", count);
    return 0;
}




